gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
-|   |  | | * **2** - wheelchair boarding is not possible at this stop |  
-|   |  | | When a stop is part of a larger station complex, as indicated by a stop with a **parent_station** value, the stop's **wheelchair_boarding** field has the following additional semantics: |  
-|   |  | | * **0** (or empty) - the stop will inherit its **wheelchair_boarding** value from the parent station, if specified in the parent |  
-|   |  | | * **1** - there exists some accessible path from outside the station to the specific stop / platform | 
-|   |  | | * **2** - there exists no accessible path from outside the station to the specific stop / platform | 
-|   |  | | For station entrances, the **wheelchair_boarding** field has the following additional semantics: | 
-|   |  | | * **0** (or empty) - the station entrance will inherit its **wheelchair_boarding** value from the parent station, if specified in the parent |  
-|   |  | | * **1** - the station entrance is wheelchair accessible (e.g. an elevator is available to platforms if they are not at-grade)  |  
-|   |  | | * **2** - there exists no accessible path from the entrance to station platforms |  
+|  stop_id | ID | **Required** | Identifies a stop, station, or station entrance/exit. Stops, stations or station entrances/exits are collectively referred to as locations. Multiple routes may use the same stop. | 
+|  stop_code | Text | Optional | Short text or a number that identifies the location for riders. These codes are often used in phone-based transit information systems or printed on signage to make it easier for riders to get information for a particular location. The `stop_code` can be the same as `stop_id` if it is public facing. This field should be left "empty" for locations without a code presented to riders. | 
+|  stop_name | Text | **Required** | Name of a location. Use a name that people will understand in the local and tourist vernacular. | 
+|  stop_desc | Text | Optional | Description of a location that provides useful, quality information. Do not simply duplicate the name of the location. | 
+|  stop_lat | Latitude | **Required** | Latitude of a location. | 
+|  stop_lon | Longitude | **Required** | Longitude of a location. |  
+|  zone_id | ID | **Conditionally Required** | Identifies the fare zone for a stop. This field is required if providing fare information using [fare_rules.txt](#fare_rulestxt), otherwise it is optional. If this `stop_id` represents a station or station entrance/exit, the `zone_id` is ignored. |  
+|  stop_url | URL | Optional | URL of a web page about a particular location. This should be different from the `agency_url` and the `route_url` field values. | 
+|  location_type | Enum | Optional | Indicates whether this `stop_id` represents a stop, station, or station entrance/exit. If `location_type` is omitted, all locations are treated as stops. Valid values are: |  
```

Comment:
If you could us `<br>` instead of new cells, that would be amazing!

Reply:
@LeoFrachet I'm a little confused. Do you want to use line breaks instead of cells to denote all enum options?

Reply:
It's really good! In the example you're linking above, you could also add the Example of direction_id in the same cell, that would be even better.

But now that I see it formated, I do think that the "ID from calendar.service_id" are taking to much place. @barbeau @antrim Thoughts?

Reply:
I've added examples to the same cell using the <hr> tag as I think it gives them the proper visual weight. Heres a gist that shows the new enum option and example formatting. LMK what you all think.

[https://gist.github.com/giocorti/7eb24dab6c7a6546db44c23b02802d65](https://gist.github.com/giocorti/7eb24dab6c7a6546db44c23b02802d65)